setup.py

```diff
@@ -304,6 +312,8 @@ def awscrt_ext():
         # OpenBSD doesn't have librt; functions are found in libc instead.
         if not sys.platform.startswith('openbsd'):
             libraries += ['rt']
+        if using_system_libcrypto():
+            libraries += ['crypto']
```

Comment:
utterly trivial: whitespace between the openbsd stuff and this
```suggestion
        # OpenBSD doesn't have librt; functions are found in libc instead.
        if not sys.platform.startswith('openbsd'):
            libraries += ['rt']

        if using_system_libcrypto():
            libraries += ['crypto']
```